Tests/Marketplace/Tests/upload_packs_test.py

```diff
@@ -16,11 +16,13 @@
 
 class TestModifiedPacks:
     @pytest.mark.parametrize("packs_names_input, expected_result", [
-        ("pack1,pack2,pack1", {"pack1", "pack2"}),
-        ("pack1, pack2,  pack3", {"pack1", "pack2", "pack3"})
+        ('{"packs_to_upload": ["pack1", "pack2"], "packs_to_update_metadata": ["pack3"]}',
+         ({'pack1', 'pack2'}, {'pack3'})),
+        ('{"packs_to_upload": ["pack1", "pack2", "pack2"], "packs_to_update_metadata": ["pack3"]}',
+         ({'pack1', 'pack2'}, {'pack3'}))
     ])
     def test_get_packs_names_specific(self, packs_names_input, expected_result):
-        modified_packs = get_packs_ids_to_upload(packs_names_input)
```

Comment:
Since we didn't remove the `get_packs_ids_to_upload` function, let's leave a UT for it.